source/release_notes.md

```diff
@@ -31,6 +31,21 @@ The following new features and changes to the service are available.
   </tr>
 </table>
 
+{% anchor h2 %}
+9 April 2018
+{% endanchor %}
+
+{% anchor h3 %}
+Truncating fields that go over max field character limits
+{% endanchor %}
+
+<table class="table" style="width: 100%;">
+  <tr>
+    <td style="width:75px; height:75px"><img src="{{root_url}}/images/ui_icon.png" alt="An update to the UI" ><img src="{{root_url}}/images/code_icon.png" alt="An update to the API" width="100" height="100"></td>
+  <td>In the <code>POST /contactdb/lists/{list_id}/recipients/{recipient_id}</code> Add a Single Recipient to a List method, OR when you adding CSV to the UI, if you are adding or updating a <code>first_name</code>, <code>last_name</code>, or <code>email</code> fields and you go over the maximum character limit, we now automatically truncate fields to the maximum character limit so that the field is valid.</td>
```

Comment:
When adding or updating a <code>first_name</code>, <code>last_name</code>, or <code>email</code> field, SendGrid will automatically truncate any field that exceeds the maximum character limit to ensure that the field is valid. This change applies to contacts added using the UI or when using the <code>POST /contactdb/lists/{list_id}/recipients/{recipient_id}</code> Add a Single Recipient to a List method.

Reply:
Some additional points / clarification: 
- We are truncating just first and last name.
- Non-unicode characters won't be allowed on first and last name.